Documentation/toc.yml

```diff
@@ -128,7 +128,11 @@
     - name: Using the Visual Profiler
       href: Diagnostics/UsingVisualProfiler.md
   - name: Services
-    href: MixedRealityServices.md
+    items:
+    - name: What makes a mixed reality feature
+      href: MixedRealityServices.md
+    - name: What are the MixedRealityServiceRegistry and IMixedRealityServiceRegistrar
+      href: ServiceUtilities/MixedRealityServiceRegistryAndIMixedRealityServiceRegistrar.md
```

Comment:
Consider moving this content to the Architecture section above, as this does feel like Architecture-y sort of stuff (i.e. it explains system creation/setup/destruction stuff)

Reply:
i believe it is less discoverable there,